frontend/app_flowy/packages/appflowy_editor/lib/src/render/selection_menu/selection_menu_widget.dart

```diff
@@ -53,6 +53,70 @@ class SelectionMenuItem {
       editorState.apply(transaction);
     }
   }
+
+  factory SelectionMenuItem.node({
+    required String name,
+    required IconData iconData,
```

Comment:
In my opinion, the factory function makes the code a little easier to read. :)

> If using IconData here, then we can't access the selection state.

I used `IconData` because I moved
```
color: onSelected
        ? editorState.editorStyle.selectionMenuItemSelectedIconColor
        : editorState.editorStyle.selectionMenuItemIconColor,
```
inside the factory function.

I'm happy to remove the factory function, just wanted to share my opinion.

Reply:
How about adding some detailed comments above this function? It will make it more clear.